src/commands/test.ts

```diff
@@ -387,11 +388,26 @@ async function processTestScripts(
         }
 
         console.log(`Run test function ${test}`);
-        await testSuiteInstance[test]();
-        if (expectedRevertMessage !== undefined) {
-          throw new Error(
-            `Test was expected to revert with message "${expectedRevertMessage}" but no revert happened.`
-          );
+
+        const noOfFuzzingParameters = testSuiteInstance[test].length;
+        const testIterations = noOfFuzzingParameters > 0 ? 64 * noOfFuzzingParameters : 1;
+        for (let iterations = 0; iterations < testIterations; iterations++) {
+          const fuzzArguments = [];
+
+          if (noOfFuzzingParameters > 0) {
+            const randomNumbers = randomBytes(noOfFuzzingParameters * 32);
+            for (let i = 0; i < noOfFuzzingParameters; i++) {
+              fuzzArguments.push(BigInt(`0x${randomNumbers.subarray(i * 32, (i + 1) * 32).toString("hex")}`));
+            }
+            console.log(`Fuzzing with arguments`, fuzzArguments);
+          }
+
+          await testSuiteInstance[test](...fuzzArguments);
```

Comment:
We implicitly assume that the arguments of test functions are always of type `bigint` and signal that the test should get 'fuzzed'. Which is probably a fine assumption. But maybe it's worth to write that down somewhere? That to apply fuzzing to a test you need to add `bigint` arguments to a test function.

Reply:
Absolutely. There is so much documentation missing about everything regarding this tool. This is still a big topic (https://github.com/pendulum-chain/wasm-deploy/issues/8). I would not bother to add it here as the structure is quite makeshift at the moment and is going to change.